gapis/api/vulkan/extensions/khr_timeline_semaphore.api

```diff
@@ -74,25 +98,33 @@ class VkSemaphoreWaitInfoKHR {
 cmd VkResult vkSignalSemaphoreKHR(
     VkDevice                                    device,
     const VkSemaphoreSignalInfoKHR*             pSignalInfo) {
-  return VK_SUCCESS
+  signal_info := pSignalInfo[0]
+  SignalSemaphore(signal_info.semaphore, signal_info.value)
+  return ?
 }
 
 // Partial implementation
 @extension("VK_KHR_timeline_semaphore")
 @indirect("VkDevice")
+@custom
 cmd VkResult vkWaitSemaphoresKHR(
     VkDevice                                    device,
     const VkSemaphoreWaitInfoKHR*               pWaitInfo,
     u64                                         timeout) {
```

Comment:
Missing device validity check in `vkSignalSemaphore(KHR)` and `vkWaitSemaphore(KHR)`